common/meta/types/src/seq_value.rs

```diff
@@ -92,3 +93,19 @@ impl<T> SeqV<T> {
         self
     }
 }
+
+impl<T> Ord for SeqV<T>
+where T: Ord
+{
+    fn cmp(&self, other: &SeqV<T>) -> Ordering {
+        self.data.cmp(&other.data)
+    }
+}
```

Comment:
Why does SeqV need to compare by data? The data could be an arbitrary byte sequence.

Reply:
> Why does SeqV need to compare by data? The data could be an arbitrary byte sequence.

TxnCondition has some compare by value

Reply:
It's a little bit misleading, to determine the order of `SeqV` without comparing the field `seq`. 

It's hard for a user to know that `seq_v_1 < seq_v_2` only compares the `data`.
And this `Ord` will break the integrity: `!(a < b) && !(b < a)` does not imply `a == b`, unless the `a == b` is determined by only `SeqV.data`, which is still weird.